objectivec/GPBBootstrap.h

```diff
@@ -74,12 +91,16 @@
 #endif
 #endif
 
-// If property name starts with init we need to annotate it to get past ARC.
-// http://stackoverflow.com/questions/18723226/how-do-i-annotate-an-objective-c-property-with-an-objc-method-family/18723227#18723227
+/**
+ * If property name starts with init we need to annotate it to get past ARC.
+ * http://stackoverflow.com/questions/18723226/how-do-i-annotate-an-objective-c-property-with-an-objc-method-family/18723227#18723227
+ **/
 #define GPB_METHOD_FAMILY_NONE __attribute__((objc_method_family(none)))
 
-// The protoc-gen-objc version which works with the current version of the
-// generated Objective C sources.  In general we don't want to change the
-// runtime interfaces (or this version) as it means everything has to be
-// regenerated.
+/**
+ * The protoc-gen-objc version which works with the current version of the
+ * generated Objective C sources.  In general we don't want to change the
+ * runtime interfaces (or this version) as it means everything has to be
+ * regenerated.
+ **/
```

Comment:
Maybe at a \@note that this should be an internal detail to the library and is just here for the generated code to use?  (same sorta goes for GPB_METHOD_FAMILY_NONE)